components/camel-jms/src/main/java/org/apache/camel/component/jms/JmsConfiguration.java

```diff
@@ -491,6 +491,9 @@
         + " By enabling this option the message payload is read into memory in chunks and each chunk is then written to the StreamMessage until no more data.")
     private boolean streamMessageTypeEnabled;
 
+    @UriParam(label = "producer", description = "Sets whether JMS date properties should be formatted according to the ISO 8601 standard.")
+    private boolean formatDatePropertiesToIso8601;
```

Comment:
We should call it headers as that is what they are stored as in Camel, eg formatDateHeadersToIso...
And you should have this option on the JmsComponent also so you can configure it once only (eg globally).

Reply:
Thanks for input Claus. I will do so.